portfolio/src/main/webapp/script.js

```diff
@@ -99,3 +99,12 @@ function displayRandomProject() {
   updateProjectLink(project.link);
   updateProjectImage(project.image);
 }
+
+/**
+ * Fetches the hello message and adds it to the page.
+ */
+async function getHello() {
```

Comment:
maybe call it something more generic in case the message changes? `getGreeting` maybe

Reply:
I change it in the next PR to getMessages and then eventually to getComments, should I change it here still?

Reply:
That's fine, it can wait till the next PR.